cfpb/cfgov-refresh
# Making the image clickable in the image text 50 / 50

Making the image clickable in the image text 50 / 50


## Changes

- Modified  `cfgov/jinja2/v1/_includes/molecules/image-text-50-50.html` the to add the `should_link_image` flag.
- Modified  `_includes/organisms/image-text-50-50-group.html` to update the context object with the 
`should_link_image` flag.
- Modified `cfgov/jinja2/v1/_includes/organisms/image-text-50-50-group.html` to add the `should_link_image` Boolean flag.

## Testing

- Add the image text 50 / 50 group to the browse page.
- Add images, text and links to the group just added.
- Enable the `should_link_image` flag.
- Preview the page.
- Click the images within the groups, use should be redirected to the correct image.

## Screenshots

<img width="1148" alt="screen shot 2017-03-06 at 12 10 35 pm" src="https://cloud.githubusercontent.com/assets/1696212/23620753/102eaa2a-0266-11e7-9621-6f0b26cc7cc5.png">


## Notes

-

## Todos

-

## Checklist
